packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -454,6 +455,26 @@ export class CeramicCliUtils {
     } catch (e) {
       // TODO handle invalid configuration
     }
+
+    // If nothing found in default config file path, check legacy path too
+    // TODO(1615): Remove this backwards compatibility code
+    const legacyCliConfigPath = path.join(DEFAULT_CLI_CONFIG_PATH, LEGACY_CLI_CONFIG_FILE)
+    try {
+      await fs.access(legacyCliConfigPath)
```

Comment:
Arrgh, really dislike this abuse of exception handling for control flow purposes, but I know of no better way :(

Reply:
I'm sure fs has some way to check if a file is present?

Reply:
`fs.access` is richer, and serves a replacement for deprecated `fs.exists`. In addition to existence checking, it also makes sure you can read that. But its API sucks :(

Reply:
yeah, I'm also not a fan of exceptions for control flow but this code is pretty isolated so I'm not that worried about it at the moment.  There are a lot of ways this code could be cleaned up at some point, but that feels low priority at the moment